pytimize/programs/_linear_program.py

```diff
@@ -494,81 +501,103 @@ def __to_canonical_form(self, basis: List[int], show_steps: bool):
 
     def create_dual(self, show_steps: bool=True) -> "LinearProgram":
         """
-        Creates the associated duality program. 
+        Creates the corresponding dual program.
 
         Parameters
         ----------
-        show_steps : bool, optional (default=True)
-            Whether steps should be stored or not for this operation.
+        show_steps : bool (default=True)
+            Whether or not the steps should be displayed.
 
         Returns
         -------
         result : LinearProgram
-            The computed result of the method.
+            The corresponding dual program.
 
         """
-        
-#          max          |   min
-#--------------------|-------------
-#              <=   >= 0
-#   constraint  =   free variable
-#              >=   <= 0
-#
-#    variable  >=0  >=
-#              free =  
-#              <=0  <=  constraint
-
-        if not self.z == 0: # TODO remove; constants are ignored in dual
-            raise ValueError()
+        pass
+    
+        #          max          |   min
+        #--------------------|-------------
+        #              <=   >= 0
+        #   constraint  =   free variable
+        #              >=   <= 0
+        #
+        #    variable  >=0  >=
+        #              free =  
+        #              <=0  <=  constraint
 
         #LinearProgram(self._A.T, self._c, self._b, 0)
 
-
-
-    def two_phase_simplex(self, show_steps: bool=True, in_place: bool=False):
+    
+    def create_auxiliary(self, show_steps: bool=True) -> "LinearProgram":
         """
-        Computes the optimal solution for the linear program or returns a certificate of unboundedness
-        using the simplex algorithm.
+        Creates the corresponding auxiliary program. This operation requires 
+        the program to be in SEF.
 
         Parameters
         ----------
-        show_steps : bool, optional (default=True)
-            Whether steps should be stored or not for this operation.
-
-        in_place : bool, optional (default=True)
-            Whether the operation should return a copy or be performed in place.
+        show_steps : bool (default=True)
+            Whether or not the steps should be displayed.
 
         Returns
         -------
-        result : tuple (ndarray of float, LinearProgram) # TODO update to match return format
-            The copy of the linear program or self in canonical form.
+        result : "LinearProgram"
```

Comment:
Why quotes?

Reply:
It's to match the function return signature. Function return typing needs quotes for classes in order for it to be valid.